packages/engine/src/scene/functions/loaders/ModelFunctions.ts

```diff
@@ -42,6 +44,27 @@ export const deserializeModel: ComponentDeserializeFunction = (entity: Entity, d
   setComponent(entity, SceneAssetPendingTagComponent, true)
 }
 
+function setupAnimations(model, scene, entity) {
```

Comment:
what is the reason for doing this? this should be done in the existing LoopAnimationFunctions implementation